cantools/database/utils.py

```diff
@@ -337,3 +343,12 @@ def prune_database_choices(database):
         for signal in message.signals:
             prune_signal_choices(signal)
 
+
+DEFAULT: Final = 'default'
```

Comment:
this might better be a function (which does nothing) just like the other sort functions? It would simplify the typing, but on the other hand, this would make implementing different behaviors for multiple file formats complicated. You decide.

Reply:
Yes, passing in a function would have simplified typing but I think passing a function which is not executed but compared whether it is a certain object would be somewhat confusing. And now that we have `type_sort_signals` it is easy to maintain.